core-tests/pom.xml

```
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~
+  ~  Branch
+  ~  Copyright © 2021 Aurium
+  ~
+  ~  Branch is free software: you can redistribute it and/or modify
+  ~  It under the terms of the GNU Affero General Public License as
+  ~  published by the Free Software Foundation, either version 3 of the
+  ~  License, or (at your option) any later version.
+  ~
+  ~  Branch is distributed in the hope that it will be useful,
+  ~  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  ~  GNU Affero General Public License for more details.
+  ~
+  ~  You should have received a copy of the GNU Affero General Public License
+  ~  along with Branch. If not, see <https://www.gnu.org/licenses/>
+  ~  and navigate to version 3 of the GNU Affero General Public License.
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>me.aurium</groupId>
+    <artifactId>branch-core-tests</artifactId>
+    <version>0.1.0</version>
+
+    <properties>
+        <maven.compiler.source>${compiler.version}</maven.compiler.source>
+        <maven.compiler.target>${compiler.version}</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.0-alpha1</version>
```

```suggestion
            <version>1.7.30</version>
```
2.0.0 is a bit of a tough requirement. Libraries usually allow more flexibility in transitive dependency choice. slf4j 1.7 is more widely implemented by logging frameworks than 2.0.0

---

I think i moved this over to the test-module (but given your latest comment it will be moved back)

---

Yeah, this is a test only dependency so i see no point in using an older version